script.js

```diff
@@ -0,0 +1,103 @@
+const newList = document.querySelector('.input-text')
+const addItem = document.querySelector('.add-item')
+const myList = document.querySelector('.todo__item')
+const acceptItem = document.querySelector('.todo__item-name')
+
+function Item(name) {
```

Comment:
what about arrow functions?
Also, you shouldn't use `new` with `function`, because it's deprecated, we have `class` for it, but for now please skip it